subprojects/core-api/src/main/java/org/gradle/api/provider/ProviderFactory.java

```diff
@@ -146,7 +140,6 @@
      *
      * @since 6.1
      */
-    @Incubating
     FileContents fileContents(RegularFile file);
```

Comment:
The `FileContents` interface should also be promoted

Reply:
Thanks for the review, @eskatos. `FileContents` was promoted in 3003017bd82bf91b56a2db42a6b07bc91bc0508b (and later added to the release notes).